package.json

```diff
@@ -32,13 +32,13 @@
   },
   "homepage": "https://github.com/Automattic/eslint-config-wpvip#readme",
   "dependencies": {
-    "@babel/eslint-parser": "7.21.3",
+    "@babel/eslint-parser": "^7.22.10",
```

Comment:
Can we stick to exact refs?

Reply:
We can, but we have copies of `@babel/eslint-parser` in our projects (say, in vip-cli). All babel-related packages are updated together, and we will end up having multiple copies of `@babel/eslint-parser`.

Reply:
We can remove `@babel/eslint-parser` from VIP CLI, as well as any other peer dependencies leftover from older versions of this plugin. We now have a [module resolution adapter](https://github.com/Automattic/eslint-config-wpvip/blob/trunk/init.js) that pulls dependencies from this project.

I have done this for other projects I've updated and it works very well. The only two dependencies needed when using this plugin are `eslint` and `@automattic/eslint-plugin-wpvip`.

https://github.com/Automattic/eslint-config-wpvip#configuration

Reply:
OK, but if we stick to the exact version, will it be safe to update other Babel-related packages? I would like to avoid publishing a new package version every time we update Babel.

Reply:
If we align on major version (`7`, currently) then I have not seen any issues with minor version drift.

Reply:
OK, I have removed the caret.